src/pages/User/User.js

```diff
@@ -59,44 +60,48 @@ class User extends React.Component {
-                {data.instagram_username && (
-                  <InstagramUser
-                    href={`https://www.instagram.com/${data.instagram_username}`}
-                    target="_blank"
-                  >
-                    @{data.instagram_username}
-                  </InstagramUser>
-                )}
-                <StatsContainer>
-                  <StyledDiv>
-                    <StyledNumbers>
-                      {convertedNumbers(data.downloads)}
-                    </StyledNumbers>
-                    <div>Downloads</div>
-                  </StyledDiv>
-                  <StyledDiv>
-                    <StyledNumbers>
-                      {convertedNumbers(data.followers_count)}
-                    </StyledNumbers>
-                    <div>Followers</div>
-                  </StyledDiv>
-                  <StyledDiv>
-                    <StyledNumbers>
-                      {convertedNumbers(data.following_count)}
-                    </StyledNumbers>
-                    <div>Following</div>
-                  </StyledDiv>
-                </StatsContainer>
-              </UserContainer>
-              <Collections username={username} />
-              <Photos username={username} />
-            </>
+          {!error ? (
+            readyToLoad && (
+              <>
+                <UserContainer>
+                  <StyledImage src={data.profile_image.large} alt={data.name} />
+                  <UserName>{data.name}</UserName>
+                  {data.bio && <Bio>{data.bio}</Bio>}
+                  {data.instagram_username && (
+                    <InstagramUser
+                      href={`https://www.instagram.com/${data.instagram_username}`}
+                      target="_blank"
+                    >
+                      @{data.instagram_username}
+                    </InstagramUser>
+                  )}
+                  <StatsContainer>
+                    <StyledDiv>
+                      <StyledNumbers>
+                        {convertedNumbers(data.downloads)}
+                      </StyledNumbers>
+                      <div>Downloads</div>
+                    </StyledDiv>
+                    <StyledDiv>
+                      <StyledNumbers>
+                        {convertedNumbers(data.followers_count)}
+                      </StyledNumbers>
+                      <div>Followers</div>
+                    </StyledDiv>
+                    <StyledDiv>
+                      <StyledNumbers>
+                        {convertedNumbers(data.following_count)}
+                      </StyledNumbers>
+                      <div>Following</div>
+                    </StyledDiv>
+                  </StatsContainer>
+                </UserContainer>
+                <Collections username={username} />
+                <Photos username={username} />
+              </>
+            )
+          ) : (
+            <NotFound errorMessage={errorMessage} />
```

Comment:
so In stead of conditionally rendering the not found component, you could use this.props.history.pus('/not-found'), we can discuss tomorrow? remind me please